libraries/AP_HAL_ChibiOS/hwdef/common/board.c

```diff
@@ -238,11 +238,8 @@ void __late_init(void) {
  * @brief   SDC card detection.
  */
 bool sdc_lld_is_card_inserted(SDCDriver *sdcp) {
-  static bool last_status = false;
-
-  if (blkIsTransferring(sdcp))
-    return last_status;
-  return last_status = (bool)palReadPad(GPIOC, 11);
+    (void)sdcp;
```

Comment:
Why change this behavior?

Reply:
because it was wrong - it is a specific pin in generic code. I suspect it was cut&pasted in the original implementation